storage/functions.go

```diff
@@ -16,10 +17,9 @@ var ErrResourcesMissing = errors.New("This resources is missing or old value is
 
 const addResourceQuery = `
 	INSERT INTO 
-	resources 
-	(id, category, content) 
+	resources(id, category, content) 
 	VALUES 
-	(UUID_TO_BIN(?), ?, ?)
+	(UUID_TO_BIN(?), ?, CAST(CONVERT(? USING utf8) AS JSON))
```

Comment:
Why do we need this casting and converting?

Reply:
To resolve `cannot create a JSON value from a string with CHARACTER SET 'binary'`
see: https://stackoverflow.com/questions/38078119/mysql-5-7-12-import-cannot-create-a-json-value-from-a-string-with-character-set